src/mixins/ready-mixin.spec.ts

```diff
@@ -1,18 +0,0 @@
-#!/usr/bin/env -S node --no-warnings --loader ts-node/esm
-
```

Comment:
Please keep this unit test file with the unit test in it for future usage.

It's a good practice for pair every source code file with a unit test `.spec.ts` file.

Reply:
OK. I saw misc-mixin has no unit-test file and thought it was not necessary for mixins with no protected property.

Reply:
I'd also like to accept a PR to add the unit test (even an empty one) to those source code files that have no `.spec.ts`.

Reply:
Sure thing, I'll try that when I have better understanding of puppet code.